community/blender/build

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -e
+
+patch -p1 < 0001-musl-fixes.patch
+patch -p1 < 0002-fix-linking-issue.patch
+patch -p1 < 0003-increase-thread-stack-size-for-musl.patch
```

Comment:
For multiple patches, it is encouraged to use 
```shell 
for patch in *.patch; do
    patch -p1 < "$patch"
done
```

Reply:
I usually do this when using more than four patches, because the line count now would be the same anyways.